plugins/mcreator-localization/help/default/villagerprofession/action_sound.md

```diff
@@ -0,0 +1 @@
+Villager sound when he uses his profession block.
```

Comment:
he -> they, English is generally gender neutral, so are villagers

Reply:
Done!